doc/api/n-api.md

```diff
@@ -2362,6 +2362,19 @@ This API allocates a JavaScript value with external data attached to it. This
 is used to pass external data through JavaScript code, so it can be retrieved
 later by native code using [`napi_get_value_external`][].
 
+**Some runtimes other than Node.js have dropped support for external buffers**.
```

Comment:
This comment seems to be out-of-place - it is currently under `napi_create_external` section.
Should we copy it instead to both `napi_create_external_arraybuffer` and `napi_create_external_buffer`?

Reply:
Thanks for catching that, I'm not sure how I put it there :(